scripts/searchApiConsumer.js

```diff
@@ -53,6 +53,15 @@ exports.fetchStilling = async (uuid) => {
         'categoryList',
         'created',
         'createdBy',
+        'employer.id',
+        'employer.uuid',
+        'employer.mediaList',
+        'employer.contactList',
+        'employer.createdBy',
+        'employer.updatedBy',
+        'employer.created',
+        'employer.updated',
+        'employer.location',
```

Comment:
Skal ikke arbeidsgivers adresse vises lengre?

Reply:
Hei, `employer.location` har ikke tidligere vært tilgjengelig. Den blir tilgjengelig nå i prod, etter at reindeksering er ferdig (et par timer til) :). 

Man har brukt en eller annen property for dette tidligere.

Reply:
(Og denne endringen er litt på siden av selve PR-en, men jeg tok det med i samme slengen.)

Reply:
Joo, vi bruker den til å vise arbeidsgivers adresse nå, så vidt jeg kan skjønne? hvis du søker opp employer.location i https://github.com/navikt/pam-stillingsok/blob/master/src/common/getEmployer.js

Reply:
Ja, jeg husker vi pratet om den, men trodde ikke den var tatt i bruk i frontend. Feltet `employer.location` har altså ikke eksistert på ads i ElasticSearch før i dag :). 
Regner med det brukes for "Om arbeidsgiver". Det er ca. litt over 100 nylig oppdaterte stillinger som har fått location, her er én:
https://arbeidsplassen.nav.no/stillinger/stilling/8e8c699c-f784-463b-9d9f-70e45320264b

Jeg skal fjerne excluden.